test/MLBridgeTest.cpp

```diff
@@ -174,53 +161,67 @@ void increment_port(int delta) {
 }
 
 int testGRPC() {
+#define gRPCModelRunnerInit(datatype)                                          \
+  increment_port(1);                                                           \
+  MLBridgeTestgRPC_##datatype::Reply response;                                 \
+  MLBridgeTestgRPC_##datatype::Request request;                                \
+  auto MLRunner = std::make_unique<                                            \
+      gRPCModelRunner<MLBridgeTestgRPC_##datatype::MLBridgeTestService,        \
+                      MLBridgeTestgRPC_##datatype::MLBridgeTestService::Stub,  \
+                      MLBridgeTestgRPC_##datatype::Request,                    \
+                      MLBridgeTestgRPC_##datatype::Reply>>(                    \
+      server_address, &request, &response, nullptr);                           \
+  MLRunner->setRequest(&request);                                              \
+  MLRunner->setResponse(&response)
+
   if (server_address == "") {
     std::cerr << "Server Address must be specified via "
                  "--test-server-address=\"<ip>:<port>\"\n";
     exit(1);
   }
   {
     gRPCModelRunnerInit(int);
-    testPrimitive<int, int>("int", 11, 12);
+    testPrimitive<int, int>(MLRunner, "int", 11, 12);
```

Comment:
Should testPrimitive take three template args?

Reply:
Third one will be inferred from the function arguments.